WordPress/src/main/java/org/wordpress/android/ui/sitecreation/misc/SiteCreationTracker.kt

```diff
@@ -102,24 +91,36 @@ class SiteCreationTracker @Inject constructor(val tracker: AnalyticsTrackerWrapp
         tracker.track(AnalyticsTracker.Stat.ENHANCED_SITE_CREATION_SITE_DESIGN_SELECTED, mapOf("template" to template))
     }
 
-    fun trackSiteDesignPreviewViewed(template: String) {
-        tracker.track(
-                AnalyticsTracker.Stat.ENHANCED_SITE_CREATION_SITE_DESIGN_PREVIEW_VIEWED,
-                mapOf("template" to template)
-        )
-    }
-
-    fun trackSiteDesignPreviewLoading(template: String) {
-        tracker.track(
-                AnalyticsTracker.Stat.ENHANCED_SITE_CREATION_SITE_DESIGN_PREVIEW_LOADING,
-                mapOf("template" to template)
-        )
-    }
-
-    fun trackSiteDesignPreviewLoaded(template: String) {
-        tracker.track(
-                AnalyticsTracker.Stat.ENHANCED_SITE_CREATION_SITE_DESIGN_PREVIEW_LOADED,
-                mapOf("template" to template)
-        )
+    fun trackSiteDesignPreviewViewed(template: String?) {
+        if (template == null || template == defaultTemplateSlug) {
```

Comment:
does that mean we'll never have a `template: default` prop for those events? Makes sense to me as the slug for default could change later on 👍

Reply:
Thank you for reviewing this @Tug 🙇 

> does that mean we'll never have a template: default prop for those events?

Exactly! I aligned with iOS on this. It seemed reasonable not to pass `default`

<image src="https://user-images.githubusercontent.com/304044/100235624-51f95900-2f35-11eb-9840-12be93d6028b.png"/>